resources/views/layouts/app.blade.php

```diff
@@ -68,6 +64,8 @@
                             @endif
                         @else
                         <!-- Link to Restaurant page -->
+                        <li class="nav-item nav-item my-auto me-3">
```

Comment:
we don't need this @Tommmy1018  because it will duplicate the menu
![image](https://github.com/kredo-school/14th_explore_ease/assets/119660809/d32be2e1-3666-4285-a277-4c74e259c39e)